README.md

```diff
@@ -4,6 +4,68 @@
 
 The New Errata Workflow Automation (NEWA) is an attempt to replace legacy testing workflow with a new one based on the use of `tmt` (Test Management Tool), Testing Farm, Jira and ReportPortal. It ensures transparency and consistency by “standardizing” the errata testing.
 
+```mermaid
+graph TB
+
+  subgraph ErrataTool
+    Node10[Event ER#123456]
+  end
+
+  subgraph Jira
+    Node20[epic - Errata ER#123456]-->Node22[task - Regression testing]
+    Node20-->Node23[task - EW checklist]
+    Node23-->Node24[subtask - Verifications]
+    Node23-->Node25[subtask - ...]
+    Node23-->Node26[subtask - Docs]
+  end
+
+  subgraph TestingFarm
+    Node30[request UUID1]
+    Node31[request UUID2]
+  end
+
+  subgraph ReportPortal
+    Node40[launch UUID1]
+  end
+
+  ErrataTool==>Jira
+  ReportPortal==>TestingFarm
+  Node22==>TestingFarm
+  ReportPortal==>Node22
+
+
+  %% Color definitions
+  classDef blue_color fill:#5765ab
+  classDef bluedark_color fill:#112170,color:#ffffff
+  classDef yellow_color fill:#d9d05b,color:#000000
+  classDef yellowdark_color fill:#e0d10d,color:#000000
+  classDef green_color fill:#49f28a,color:#000000
+  classDef greendark_color fill:#0a8f3d,color:#000000
+  classDef red_color fill:#db6969
+  classDef reddark_color fill:#9e0909,color:#ffffff
+
+  %% Colors to node mapping
```

Comment:
Also, could you maybe use lighter colors? This looks too dark. E.g.
```
  classDef blue_color fill:#d9ffff
  classDef bluedark_color fill:#00a4ff,color:#ffffff
  classDef yellow_color fill:#fff180,color:#000000
  classDef yellowdark_color fill:#ffd000,color:#000000
  classDef green_color fill:#98ff99,color:#000000
  classDef greendark_color fill:#27c329,color:#000000
  classDef red_color fill:#ffbfca
  classDef reddark_color fill:#dd2b4a,color:#ffffff
```

Reply:
OK, I updated those colors as suggested.

Here is some comparision of colors.


<img width="1149" height="769" alt="mermiad_dark_after" src="https://github.com/user-attachments/assets/f1a97a65-1d7c-4c7b-869d-4995f6ba28ff" />
<img width="1149" height="769" alt="mermiad_light_after" src="https://github.com/user-attachments/assets/28f6c208-8891-4fb6-8ebf-43f48e5d88b1" />
<img width="1144" height="753" alt="mermiad_light_before" src="https://github.com/user-attachments/assets/df078664-4b6f-4baa-83c5-c271d82523d6" />
<img width="1144" height="753" alt="mermiad_dark_before" src="https://github.com/user-attachments/assets/7135943d-e8d0-4035-adb6-f10593bcaec4" />